SCClassLibrary/Common/Audio/SynthDef.sc

```diff
@@ -541,9 +541,13 @@ SynthDef {
 	}
 
 	// make SynthDef available to all servers
-	add { arg libname, completionMsg, keepDef = true;
-		var	servers, desc = this.asSynthDesc(libname ? \global, keepDef);
-		if(libname.isNil) {
+	add { | libname = \sendToEverything, completionMsg, keepDef = true |
+		var servers;
+		this.asSynthDesc(libname ? \global, keepDef);
+		if(libname.isNil || (libname == \sendToEverything)) {
+			if (libname.isNil) {
+				"Provided server is nil, SynthDef may not be sent".warn;
```

Comment:
The purpose of this PR is to distinguish providing no argument and (accidentally) explicitly passing ```nil```. This lets us warn on ```nil```.